src/com/company/lambdaexamples/LambdaExpressions3.java

```diff
@@ -0,0 +1,90 @@
+package com.company.lambdaexamples;
+
+import java.util.*;
+
+public class LambdaExpressions3 {
+    public static void main(String[] args) {
+        List<Integer> intNumbers = new ArrayList<>();
+        intNumbers.add(100);
+        intNumbers.add(-999);
+        intNumbers.add(-9);
+        intNumbers.add(8);
+        intNumbers.add(80);
+        taskOne(intNumbers);
+        Person firstPerson = new Person("Iliana Markova", 165);
+        Person secondPerson = new Person("Test Testov", 180);
+        Person thirdPerson = new Person("LoremLoremLorem  LoremLoremLorem", 155);
+        List<Person> listPerson = new ArrayList<>();
+        listPerson.add(firstPerson);
+        listPerson.add(secondPerson);
+        listPerson.add(thirdPerson);
+        addPersonNames(listPerson);
+        getLongNameCharacter(listPerson);
+        getHighestPerson(listPerson);
+        getSquareNumbers(intNumbers);
+    }
+
+
+//1. Направете метод, който за List от Integer-и, да намира дали има число, което е трицифрено.
+
+    public static void taskOne(List<Integer> listNumbers) {
+        listNumbers.forEach((n) -> {
```

Comment:
You can use stream API like this:
`listNumbers.stream().anyMatch(n -> ((n >= 100 && n <= 999) || (...)); `
it's just another example of the stream API

Reply:
code block is changed as it is suggested